.travis.yml

```
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk8
 before_script:
   - wget https://dist.ipfs.io/go-ipfs/v0.4.16/go-ipfs_v0.4.16_linux-amd64.tar.gz -O /tmp/go-ipfs_linux-amd64.tar.gz
```

We should switch to latest go-ipfs while we are at it:
```suggestion
  - wget https://dist.ipfs.io/go-ipfs/v0.5.1/go-ipfs_v0.5.1_linux-amd64.tar.gz -O /tmp/go-ipfs_linux-amd64.tar.gz
```

---

This will [break](https://github.com/ipfs-shipyard/java-ipfs-http-client/issues/157) the `/api/v0/get` endpoint, and possibly some other ones as well.